src/components/layouts/hero/hero.tsx

```diff
@@ -20,7 +20,7 @@ interface PropTypes {
 const Hero = ({ title, description, image, buttons }: PropTypes) => (
   <div className={cn(hero)}>
     <Container fluid={true} alignItems={'center'}>
-      <Grid lg={4} md={3} sm={12} xs={12} className={cn(pb32)}>
+      <Grid lg={6} md={7} sm={12} xs={12} className={cn(pb32)}>
```

Comment:
```suggestion
      <Grid lg={5} md={7} sm={12} xs={12} className={cn(pb32)}>
```
I see why this is intended to fill half of the available space, but I disagree with doing so, as this decreases space around the text too much. Leaving it at 5 gives the content room to breathe

Reply:
```suggestion
      <Grid lg={6} md={6} sm={12} xs={12} className={cn(pb32)}>
```

> I see why this is intended to fill half of the available space, but I disagree with doing so, as this decreases space around the text too much. Leaving it at 5 gives the content room to breathe

the same reasoning applies here, although there is less real estate to begin with. I believe a 6/6 split works best here, and images are still displayed at a reasonable size for big tablets / small laptops